docker-image-src/4.0/docker-entrypoint.sh

```diff
@@ -428,16 +419,16 @@ for conf in ${!COMMUNITY[@]} ; do
     fi
 done
 
-for conf in ${!ENTERPRISE[@]} ; do
-
-    if [ "${NEO4J_EDITION}" == "enterprise" ];
-    then
-       if ! grep -q "^$conf" "${NEO4J_HOME}"/conf/neo4j.conf
-       then
-        echo -e "\n"$conf=${ENTERPRISE[$conf]} >> "${NEO4J_HOME}"/conf/neo4j.conf
-       fi
-    fi
-done
+#for conf in ${!ENTERPRISE[@]} ; do
```

Comment:
Same as the above - if the array is empty then maybe we could keep this uncommented, as its a no-op?